substrate/zombienet/0001-basic-warp-sync/README.md

```diff
@@ -32,33 +33,30 @@ The zombienet shall be executed with the following command, and run for some per
 ```
 
 Once the zombienet is stopped, the database snapshot
-(`{alice,bob}/data/chains/local_testnet/db/` dirs) was created using the following
+(`alice/data/chains/local_testnet/db/` dir) was created using the following
 commands:
 ```bash
-mkdir -p db-snapshot/{alice,bob}/data/chains/local_testnet/db/
+mkdir -p db-snapshot/alice/data/chains/local_testnet/db/
 cp -r db-test-gen/alice/data/chains/local_testnet/db/full db-snapshot/alice/data/chains/local_testnet/db/
-cp -r db-test-gen/bob/data/chains/local_testnet/db/full   db-snapshot/bob/data/chains/local_testnet/db/
 ```
 
-The file format should be `tar.gz`. File shall contain `local_testnet` folder and its subfolders, e.g.:
+Sample command to prepare archive:
 ```
-$ tar tzf chains.tgz | head
-local_testnet/
-local_testnet/db/
-local_testnet/db/full/
-...
-local_testnet/db/full/000469.log
+tar -C db-snapshot/alice/ -czf chains.tgz ./
 ```
 
-Sample command to prepare archive:
+The file format should be `tar.gz`. File shall contain `local_testnet` folder and its subfolders, e.g.:
 ```
-tar -C db-snapshot/alice/data/chains/ -czf chains.tgz local_testnet
+$ tar tzf chains.tgz | head
+data/chains/local_testnet/
+data/chains/local_testnet/db/
+data/chains/local_testnet/db/full/
+...
+data/chains/local_testnet/db/full/000469.log
 ```
 
 Also refer to: [zombienet#578](https://github.com/paritytech/zombienet/issues/578)
 
-The `raw` chain-spec shall also be saved: `db-test-gen/gen-db-raw.json`.
```

Comment:
We still need to keep raw chain-spec, right? I think keeping this note maybe worthy for future maintainers.